time_weighted_average_price/on_chain/Makefile

```diff
@@ -2,12 +2,6 @@
 test-local:
 	npx hardhat test --grep "Local"
 
-.PHONY: copy-from-twap
```

Comment:
I don't have this copy in the hello world on chain. I figure I'd remove that here too to keep things simple